network/network/src/main/java/bisq/network/p2p/services/confidential/resend/ResendMessageService.java

```diff
@@ -0,0 +1,177 @@
+package bisq.network.p2p.services.confidential.resend;
```

Comment:
Nit. A few added files are missing the license.

Reply:
Ups, thanks. Fixed.